docs/sources/flow/reference/components/beyla.ebpf.md

```diff
@@ -0,0 +1,250 @@
+
+{{< admonition type="note" >}}
+To run this component, the Agent requires administrative (`sudo`) privileges, or at least it needs to be granted the `CAP_SYS_ADMIN` capability.
+{{< /admonition >}}
+
+[Grafana Beyla]: https://github.com/grafana/beyla
+
+## Usage
+
+```river
+beyla.ebpf "LABEL" {
+ 
+}
+```
+
+## Arguments
+
+`beyla.ebpf` supports the following arguments:
+
+Name | Type | Description                                               | Default | Required
+---- | ---- |-----------------------------------------------------------| ------- | --------
+`open_port` | `string` | The port of the running service for Beyla automatically instrumented with eBPF. | | no
+`excutable_name` | `string` | The name of the executable to match. | | no
+
+
+`open_port` accepts a comma-separated list of ports (for example, 80), and port ranges (for example, 8000-8999). If the executable matching only one of the ports in the list, it is considered to match the selection criteria.
+
+`excutable_name` accepts a regular expression to be matched against the full executable command line, including the directory where the executable resides on the file system.
+
+## Blocks
+
+The following blocks are supported inside the definition of `beyla.ebpf`:
+
+Hierarchy | Block | Description | Required
+--------- | ----- | ----------- | --------
+routes | [routes][] | Configures the routes to match HTTP paths into user-provided HTTP routes. | no
+attributes | [attributes][] | Configures the attributes for the component. | no
+attributes > kubernetes | [kubernetes][] | Configures decorationg of the metrics and traces with Kubernetes metadata of the instrumented Pods. | no
+discovery | [discovery][] | Configures the discovery for instrumentable processes matching a given criteria. | no
+discovery > services | [services][] | Configures the discovery for the component. | no
+output | [output][] | Configures where to send received telemetry data. | yes
+
+The `>` symbol indicates deeper levels of nesting. For example,
+`attributes > kubernetes` refers to a `kubernetes` block defined inside an
+`attributes` block.
+
+[routes]: #routes-block
+[attributes]: #attributes-block
+[kubernetes]: #kubernetes-block
+[discovery]: #discovery-block
+[services]: #services-block
+[output]: #output-block
+
+### attributes block
+
+Allows configuring how some attributes for metrics and traces are decorate.
+
+Contains the following blocks:
+[kubernetes]: #kubernetes-block
+
+#### kubernetes block
+
+If you run this in a Kubernetes environment, you can configure it to decorate the traces and metrics with the Standard OpenTelemetry labels:
+
+- `k8s.namespace.name`
+- `k8s.deployment.name`
+- `k8s.statefulset.name`
+- `k8s.replicaset.name`
+- `k8s.daemonset.name`
+- `k8s.node.name`
+- `k8s.pod.name`
+- `k8s.pod.uid`
+- `k8s.pod.start_time`
```

Comment:
nit: for consistency with how we document components, it's a bit weird for all this information to be before the table. Should this be moved down to after the table?